supabase/migrations/20260816201305_channel_overrides_require_membership.sql

```
@@ -0,0 +1,134 @@
+      FROM public.role_bindings
+      WHERE role_bindings.principal_type = public.rbac_principal_apikey()
+        AND role_bindings.principal_id = p_principal_id
+        AND role_bindings.scope_type = public.rbac_scope_org()
+        AND role_bindings.org_id = p_org_id
+        AND (role_bindings.expires_at IS NULL OR role_bindings.expires_at > now())
+    )
+    ELSE false
+  END;
+$$;
+
+ALTER FUNCTION public.channel_override_principal_in_org(text, uuid, uuid) OWNER TO postgres;
```

SQLFluff (4.3.0)

[error] Line is too long (92 > 80).

(LT05)